Sources/_RegexParser/Regex/Printing/PrettyPrinter.swift

```diff
@@ -46,33 +54,37 @@ extension PrettyPrinter {
     self.minBottomUpLevels = minBottomUpLevels
   }
 
-  /// Output a string directly, without termination, without
-  /// indentation, and without updating _any_ internal state.
+  /// Outputs a string directly, without termination or
+  /// indentation, and without updating any internal state.
   ///
   /// This is the low-level interface to the pret
```

Comment:
QUERY: What was this supposed to say?  The comment looks like it got cut off mid-sentence.

Reply:
It looks like: "This is the low-level interface to the pretty printer."

Reply:
Thanks; fixed in 96df14d.